lib/core/Canvas.js

```diff
@@ -327,14 +332,10 @@ Canvas.prototype._clear = function() {
   delete this._cachedViewbox;
 };
 
-Canvas.prototype._focus = function() {
+Canvas.prototype.focus = function() {
```

Comment:
Great to have a public API for this. 

I very naturally want to control the focus state, same as in CodeMirror or any other editor.

Reply:
Added some JSDoc and generated types.